src/sentry/models/grouphashmetadata.py

```diff
@@ -53,6 +76,9 @@ class GroupHashMetadata(Model):
     # When the grouphash was created. Will be null for grouphashes created before we started
     # collecting metadata.
     date_added = models.DateTimeField(default=timezone.now, null=True)
+    # The version of the metadata schema which produced the data. Useful for backfilling when we add
+    # to or change the data we collect and want to update existing records.
+    schema_version = models.CharField(null=True)
```

Comment:
Are we planning to remove the null once all records are populated?

Reply:
it is probably not worth it as we'd have to be careful about the migration https://develop.sentry.dev/backend/application-domains/database-migrations/#adding-constraints-to-columns-including-not-null

Reply:
Agreed. It'd be nice if postgres had some sort of progressively-non-null designation we could set - so that the constraint could be checked as records are touched in the course of normal operations - but AFAIK no such thing exists, so we'll probably have to just live with the nullability. (This goes for pretty much all the fields in the table.)